app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class='container'>
+  
+  <div class='form_container'>
+  <h2>Sign up</h2>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+    
+    <div class="field">
+    <%= f.label :fullName %><br />
+    <%= f.text_field :name, autofocus: true %>
+  </div>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password_confirmation %><br />
```

Comment:
- [ ] I am able to start your app locally. Good job so far. Unfortunately I am able to Sign Up. I get the error in the screenshot below.
![image](https://github.com/souad988/budget_tracker_app/assets/62849872/e96b36dd-5d76-4006-b7ab-a2d5f9890dc9)

- The message "Unpermitted parameter: :password_confirmation" typically arises in the context of a Rails application using the Devise gem for user authentication.

This message indicates that the parameter "password_confirmation" is not permitted or allowed by the controller's strong parameters configuration. Devise, by default, expects the "password_confirmation" parameter to be present when creating a new user, as it is used for password confirmation validation.

To resolve this issue, you can permit the "password_confirmation" parameter in the strong parameters. 
When I check your application_controller, you have this protected method, but it is missing password_confirmation

**Your code**
 ```
 protected

  def update_allowed_parameters
    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password) }
    devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :email, :password, :current_password) }
  end
```
- As per the screenshot below add password_confirmation as a permitted parameter in line 37
![image](https://github.com/souad988/budget_tracker_app/assets/62849872/e9059149-7c07-4da0-b987-11f0ac9ff0de)